apps-rendering/src/quizAtom.ts

```diff
@@ -49,6 +49,7 @@ type Question = {
 	text: string;
 	answers: Answer[];
 	imageUrl?: string;
+	imageAlt?: string;
```

Comment:
We are making this change only in `apps-rendering`. Is this intentional? Do we need to do the same in DCR? : https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/components/KnowledgeQuizAtomWrapper.importable.tsx#L17

Reply:
Good catch, I also made the change here: https://github.com/guardian/dotcom-rendering/blob/dff4769950e501ce80e710d00551c2877b571df0/dotcom-rendering/src/lib/content.d.ts#L769 which seemed to be enough, but I've also made it in the wrappers now as well :)